pytorch_lightning/callbacks/pt_callbacks.py

```diff
@@ -121,10 +122,10 @@ def on_epoch_end(self, epoch, logs=None):
         current = logs.get(self.monitor)
         stop_training = False
         if current is None:
-            logging.info(
-                'Early stopping conditioned on metric `%s` '
-                'which is not available. Available metrics are: %s' %
-                (self.monitor, ','.join(list(logs.keys()))), RuntimeWarning)
+            warnings.warn(
+                f'Early stopping conditioned on metric `{self.monitor}` '
```

Comment:
it is recommended not having white space on the end of the line but rather on the beginning of the following one...

Reply:
lol it passed flake8. but I'll change this.